### PR TITLE
Corrected particles

### PR DESCRIPTION
**Pull request by mathiascode**
I wrote the wrong IDs in my previous PR, and included uppercase letters, causing the particles to become unknown to the server.

### CODE REVIEW

**Comment by madmaxoft** [comment]
Is it just me, or is the `IsInitialized` for the particle map never set to true, causing initialization over and over again? Not that it's this PR's fault, just an observation.

**Comment by madmaxoft** [comment]
If I were to write the code, I'd use `if (ParticleMap.empty())` and get rid of the `IsInitialized` altogether.

**Comment by sphinxc0re** [comment]
@madmaxoft Should we open an issue for that or can this be included in this PR?

**Comment by madmaxoft** [comment]
I think this PR would be a good place to fix it - it is a Particle fix, after all :)

@mathiascode can you replace the code at the top of the function (L1708 - L1710 in 1.8, similarly in all other protocols) with this:
```cpp
static std::map<AString, int> ParticleMap;
if (ParticleMap.empty())
```

**Comment by mathiascode** [comment]
Done.

File: src/Protocol/Protocol_1_9.cpp [hunk 1]
```diff
@@ -1724,9 +1724,8 @@ bool cProtocol_1_9_0::CompressPacket(const AString & a_Packet, AString & a_Compr
 
 int cProtocol_1_9_0::GetParticleID(const AString & a_ParticleName)
 {
-	static bool IsInitialized = false;
 	static std::map<AString, int> ParticleMap;
-	if (!IsInitialized)
```
Comment (sphinxc0re) [inline src/Protocol/Protocol_1_9.cpp]: Is this `IsInitialized` variable set anywhere else? Did you remove all its occurrences?

Comment (mathiascode) [inline src/Protocol/Protocol_1_9.cpp]: It also exists in WebAdmin.cpp, line 441-443

Comment (sphinxc0re) [inline src/Protocol/Protocol_1_9.cpp]: That is kinda confusing. I'm sorry @madmaxoft could you explain to me how this variable is used?

Comment (madmaxoft) [inline src/Protocol/Protocol_1_9.cpp]: A `static` variable is somewhere between a local variable (it is only accessible in the block where it is declared) and a global variable (it has one value, which stays between multiple entrances into that block). Basically, imagine this as being declared as a global variable `bool cProtocol_1_9_0::GetParticleID::Scope1::IsInitialized`. So it is only accessible in this `GetParticleID()` function, but it keeps its value from one function call to another.

Here's some code to illustrate:
```cpp
void printCounter(void)
{
  static int counter = 0;
  printf("%d\n", counter);
  counter += 1;
}

void printSequence(void)
{
  printCounter();  // Prints "0", "counter" is now 1
  printCounter();  // Prints "1", "counter" is now 2
  printCounter();  // Prints "2", "counter" is now 3
  counter = 4;  // compile error, "counter" not accessible from here
}
```

The primary purpose was supposed to be to make the `ParticleMap` initialization happen only once. The same effect can be obtained by asking the map if it's empty.

Comment (madmaxoft) [inline src/Protocol/Protocol_1_9.cpp]: So this variable has nothing to do with the one in WebAdmin :)

**Pull request merged**